src/gerbil/runtime/build-lib.scm

```diff
@@ -0,0 +1,57 @@
+(define (false . _) #f)
+
+(define (displayln . args)
+  (for-each display args)
+  (newline)
+  (force-output))
+
+(define (gerbil-build-cores) (string->number (getenv "GERBIL_BUILD_CORES" "1")))
+
+(define (read-all-as-lines port)
+  (let loop ((lines '()))
+    (let ((line (read-line port)))
+      (if (string? line)
+        (loop (cons line lines))
+        (reverse lines)))))
+
+(define (compiler options)
+  (lambda (modf)
+    (displayln "... compile " modf)
+    (let* ((proc
+            (open-process
+             (list path: (getenv "GERBIL_GSC" "gsc")
+                   arguments: `(,@options ,modf)
+                   stdout-redirection: #f)))
+           (status (process-status proc)))
+      (close-port proc)
+      (when (not (zero? status))
+        (error "Compilation error; gsc exit with nonzero status" proc)))))
```

Comment:
I believe we want to throw the status so that we can see it in the terminal if that fails.